### PR TITLE
Added custom font to Tutor Availability View

### PR DESCRIPTION
**Pull request by Mons1Oerjan**
screen shot 2017-03-15 at 11 55 55 pm

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -37,7 +37,7 @@
     <ConfirmationsSetting value="0" id="Add" />
     <ConfirmationsSetting value="0" id="Remove" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" assert-keyword="true" jdk-15="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" default="true" assert-keyword="true" jdk-15="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (ericdesj) [inline .idea/misc.xml]: are you sure this won't have any implications with the rest of the project?

Comment (Mons1Oerjan) [inline .idea/misc.xml]: I've seen it change from 1_7 to 1_8 and vice versa a couple of times in other people's PRs. I don't know why it does this.

Comment (Mons1Oerjan) [inline .idea/misc.xml]: am I 100% sure? no. I have no `knowledge` of these .idea files

**Comment by ericdesj** [comment]
⭐️ ⭐️ ⭐️ ⭐️ ⭐️ 

five star

**Pull request merged**